src/supplemental/nanolib/parquet/parquet.cc

```diff
@@ -680,6 +680,20 @@ parquet_file_queue_init(conf_parquet *conf)
 					closedir(dir);
 					return;
 				}
+				sprintf(file_path, "%s/%s", conf->dir,
+				    ent->d_name);
+
+				if (strstr(ent->d_name, "_") == NULL) {
+					if (unlink(file_path) != 0) {
+						log_error("Failed to remove file %s errno: %d",
+						    file_path, errno);
+					} else {
+						log_warn("Found a file without md5sum, "
```

Comment:
Why there will be a file without md5sum?

Reply:
parquet writes to a file by first creating a file without md5 and then renaming it to a file with md5 after the write is complete, so if nanomq quits while the file is being written, the file without md5 will be produced.